docs/guides/kernel.md

```diff
@@ -4,6 +4,8 @@ This page explains how to compile a Linux kernel with patches for T2 hardware su
 
 If your distro is not one of the distros with documentation on this Wiki, you may not need to compile a kernel yourself to get support for T2 hardware. Debian based systems can use the same kernel as described in the Ubuntu section, Arch based systems can use the same kernel as described in the Arch section, etc.
 
+Before beginning, set the directory that you want to install to.
```

Comment:
wdym? Set the directories the kernel modules and kernel image get installed to? Or `cd` to where you want to build it?